libunwind/src/Registers.hpp

```diff
@@ -1829,19 +1829,19 @@ class _LIBUNWIND_HIDDEN Registers_arm64 {
   CAPABILITIES_NOT_SUPPORTED
 #endif
 
-  uintptr_t getSP() const         { return _registers.__sp; }
-  void      setSP(uintptr_t value) { _registers.__sp = value; }
-  uintptr_t getIP() const         { return _registers.__pc; }
-  void      setIP(uintptr_t value) { _registers.__pc = value; }
-  uintptr_t getFP() const         { return _registers.__fp; }
-  void      setFP(uintptr_t value) { _registers.__fp = value; }
+  uintptr_t  getSP() const         { return _registers.__sp; }
+  void       setSP(uintptr_t value) { _registers.__sp = value; }
+  uintptr_t  getIP() const         { return _registers.__pc; }
+  void       setIP(uintptr_t value) { _registers.__pc = value; }
+  uintptr_t  getFP() const         { return _registers.__fp; }
+  void       setFP(uintptr_t value) { _registers.__fp = value; }
```

Comment:
What's up with these whitespace changes?

Reply:
They are in Silviu's changes.   However, there is room for improvement in that we should line up the bodies the way all the other Register classes do.  All the other classes use two space indent between type and 'getSP' though uintptr_t is one more character than uint32_t and uint64_t.  RISC-V needs a minor cleanup here as it's not quite right after being CHERI-fied.